packages/components/src/styles/leadtext/leadtext.declaration.ts

```diff
@@ -0,0 +1,19 @@
+import type { Style } from '../solid-styles.declaration';
+
+export default {
+  styleName: 'sd-leadtext',
+  summary: 'Leadtext is used for text that should be highlighted and a focal point of the page.',
+  status: 'experimental',
+  since: '1.1',
+  attributes: [
+    {
+      name: 'sd-leadtext--size-...',
+      options: ['-', 'lg'],
```

Comment:
do we need an option here if we only have one option? 

if this is anyway not showing an option here, I am fine (did not checkout the branch and tested it, but came to my mind).

Reply:
Yah, we need a second option to deactivate it again. It's just some logic for the helper. But I made it programmatically now.